northstar/src/runtime/cgroups.rs

```diff
@@ -129,13 +132,13 @@ impl CGroups {
         );
 
         let resources = cgroups_rs::Resources {
-            memory: config.memory.clone().unwrap_or_default(),
+            memory: config.memory.clone().map(Into::into).unwrap_or_default(),
```

Comment:
I see a lot of `.map(Into::into)` in this file. We have already a dependency to `itertools` that has a `.map_into()` not sure if that would improve anything tho :man_shrugging:

Reply:
In that case its an ordinary option and not an iterator. Think converting the option into an iterator for getting `Itertools::map_into` doesn't sound like a obvious improvement.

Reply:
yeah, wasn't suggesting that :sweat_smile: just thought all of these were iterators, never mind then :)